canopy/app/src/App.scss

```diff
@@ -20,8 +20,45 @@
   align-items: center;
   justify-content: center;
 
+  .side-nav {
+    background: lightgray;
+    position: fixed;
+    left: 0;
+    height: 100vh;
+    width: 10rem;
+  }
+
+  .notification-list {
+    border-right: 1px solid gray;
+    border-left: 1px solid gray;
+    background: white;
+    position: fixed;
+    left: 10rem;
+    width: 20rem;
+    height: 100vh;
+    z-index: 999;
+  }
+
   .view {
+    position: fixed;
+    left: 10rem;
     flex: 0 1 100%;
     min-height: 100vh;
   }
 }
+
+.display-flex {
+  display: flex;
```

Comment:
Single declarations shouldn't be made into classes.

There is no reason not to include these as style declarations in the necessary components.